atom/browser/lib/bluetooth_chooser.cc

```diff
@@ -95,15 +95,15 @@ void BluetoothChooser::AddOrUpdateDevice(const std::string& device_id,
                                          bool is_gatt_connected,
                                          bool is_paired,
                                          int signal_strength_level) {
-  DeviceInfo info = {device_id, device_name};
+  DeviceInfo info{device_id, device_name};
   device_list_.push_back(info);
 }
 
 void BluetoothChooser::RemoveDevice(const std::string& device_id) {
   for (auto it = device_list_.begin(); it != device_list_.end(); ++it) {
     if (it->device_id == device_id) {
       device_list_.erase(it);
-      return;
+      break;
```

Comment:
Using `return` or `break` does not matter here. Personally I think using `return` is better here, because `break` usually implies there are code after the loop.